generation/generate/generate_serialize.ts

```diff
@@ -31,7 +31,7 @@ export function generateSerialize(analysisResult: AnalysisResult): string {
         const structFields = struct.fields.filter(f => !getIsForImpl(analysisResult, f.type) && f.name !== "span");
 
         writer.writeLine("#[derive(Serialize)]");
-        writer.writeLine(`#[serde(rename = "${struct.name}", rename_all = "camelCase", tag = "nodeKind")]`);
+        writer.writeLine(`#[serde(rename = "${struct.name}", rename_all = "camelCase", tag = "kind")]`);
```

Comment:
I prefer "kind" for this because it's what the TypeScript compiler uses and the rust code. I've renamed some of the other properties above, which also helps prevent overlap in the rust code.

Reply:
I had thought so too but I found some nodes had their own "kind" field, like [PrivateMethod](https://swc.rs/rustdoc/swc_ecma_ast/struct.PrivateMethod.html). So I'm worried that renaming `nodeKind` to `kind` would cause conflicts with such existing fields.

Reply:
Yeah, I found that too. I ended up renaming those to names like `method_kind()`/`methodKind`.

Reply:
Sounds good!